### ISSUE TITLE
Improve exception handling for import errors in evaluation script

### ISSUE THREAD

**Issue by deshraj** [Cloud-CV/EvalAI#1789]
Currently if there is a challenge evaluation script which contains some custom python package requirement which is not installed on EvalAI server, then the submission worker process gets killed automatically. Please see the example log below where `tqdm` is missing in the system.

```
worker_1  | Traceback (most recent call last):
worker_1  |   File "/usr/local/lib/python2.7/runpy.py", line 174, in _run_module_as_main
worker_1  |     "__main__", fname, loader, pkg_name)
worker_1  |   File "/usr/local/lib/python2.7/runpy.py", line 72, in _run_code
worker_1  |     exec code in run_globals
worker_1  |   File "/code/scripts/workers/submission_worker.py", line 535, in <module>
worker_1  |     main()
worker_1  |   File "/code/scripts/workers/submission_worker.py", line 516, in main
worker_1  |     load_active_challenges()
worker_1  |   File "/code/scripts/workers/submission_worker.py", line 234, in load_active_challenges
worker_1  |     extract_challenge_data(challenge, phases)
worker_1  |   File "/code/scripts/workers/submission_worker.py", line 215, in extract_challenge_data
worker_1  |     challenge_module = importlib.import_module(CHALLENGE_IMPORT_STRING.format(challenge_id=challenge.id))
worker_1  |   File "/usr/local/lib/python2.7/importlib/__init__.py", line 37, in import_module
worker_1  |     __import__(name)
worker_1  |   File "/tmp/tmpm1P8t1/compute/challenge_data/challenge_59/__init__.py", line 1, in <module>
worker_1  |     from .main import evaluate
worker_1  |   File "/tmp/tmpm1P8t1/compute/challenge_data/challenge_59/main.py", line 37, in <module>
worker_1  |     from tqdm import *
worker_1  | ImportError: No module named tqdm
evalai_worker_1 exited with code 1
```

We want to make the system more robust. So, we would want to add exception handling so that the worker process doesn't get killed whenever there is a missing requirement.

**Comment by party98** [comment]
Should we catch an `ImportError` exception and extract the package name that is missing and try to install it in the submission folder using pip?
If this fails, then we can throw an error indicating to that the server was unable to run the script due to error in import the package.

**Comment by party98** [comment]
Please give a go ahead if you think this is a good idea so that I can begin work. 😀

**Comment by RishabhJain2018** [comment]
Fixed in #1801 . Hence closing this one! @party98 Please pick any other issue.
